VideoApp/VideoApp/Video/Tracks/Camera/CameraConfigFactory.swift

```diff
@@ -23,17 +23,25 @@ struct CameraConfig {
 
 class CameraConfigFactory {
     private let appSettingsStore: AppSettingsStoreWriting = AppSettingsStore.shared
+    private let remoteConfigStore: RemoteConfigStoreReading = RemoteConfigStoreFactory().makeRemoteConfigStore()
     
     func makeCameraConfigFactory(captureDevice: AVCaptureDevice) -> CameraConfig {
         var targetSize: CMVideoDimensions {
+            // 1024 x 768 squarish crop (1.25:1) on most iPhones. 1280 x 720 squarish crop (1.25:1) on the iPhone X
+            // and models that don't have 1024 x 768.
+            let hdDimensions = CMVideoDimensions(width: 900, height: 720)
+            
+            switch remoteConfigStore.roomType {
+            case .peerToPeer: return hdDimensions
+            case .group, .groupSmall, .unknown: break
+            }
```

Comment:
I think I'm going to have to confirm with @aaalaniz that this logic is correct because I had to make some assumptions. But any changes should be pretty small.

Reply:
Talked offline and Aaron is good with this.